projects/media-viewer/src/lib/media-viewer/annotation/annotation-set.service.ts

```diff
@@ -0,0 +1,31 @@
+import {Injectable} from '@angular/core';
+import {Annotation, AnnotationSet} from './annotation-set.model';
+
+
+interface IAnnotationSetService {
+  getAnnotationSet(annotationSetId: string): AnnotationSet;
+
+  save(annotationSet: AnnotationSet): void;
+
+  delete(annotation: Annotation): void;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AnnotationSetService implements IAnnotationSetService {
```

Comment:
again, it seems like the class and interface should be separated so that we export the interface and not the class, or at least that would be done independently